versioned_docs/version-8.2/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -89,7 +89,20 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
-The following sections are only needed if you are updating to v8.0.13 or the versions after v8.0.13.
+### v8.2.3
+
+:::caution Breaking Change
+Zeebe Gateway authentication is now enabled by default.
+:::
+
```

Comment:
Following the same style as other versions on this page, we need an H3 title here:

```
### Zeebe Gateway
```